npe2/_from_npe1.py

```diff
@@ -481,7 +485,7 @@ def _write_new_setup_cfg_ep(info: PluginPackage, mf_name: str):
     mf_path = f"{info.top_module}:{mf_name}"
     new_ep = f"\n{info.package_name} = {mf_path}"
     if "options.entry_points" not in p.sections():
-        p.add_section("options.entry_points")
+        p.add_section("options.entry_points")  # pragma: no cover
     p.set("options.entry_points", NPE2_EP, new_ep)
     if "options.package_data" not in p.sections():
         p.add_section("options.package_data")
```

Comment:
```suggestion
    def _mksection(p,name:str):
        if name not in p.sections():
            p.add_section(name)
    _mksection(p,"options.entry_points")
    p.set("options.entry_points", NPE2_EP, new_ep)
    _mksection(p,"opstions.package_data")
```

Reply:
I left another review comment referencing this suggestion. I don't really think it's worth adding the function to get rid of the pragma...